### PR TITLE
feat(icon): add vercel folder support

### PR DESCRIPTION
**Pull request by Hacksore**
- Add vercel folder support 
- Rename file icon `file_type_zeit.svg` -> `file_type_vercel.svg`

**Changes proposed:**

- [x] Add

Example:
image

### CODE REVIEW

**Comment by robertohuertasm** [comment]
The background color of the folder doesn't work for dark themes. Consider using another color. Grey would work for light and dark themes and it also works with the icon.

**Comment by Hacksore** [comment]
@robertohuertasm Here they are side by side with the default theme in a crude mock up. If you think black is not feasible can you suggest the ideal color for the gray?

image

**Comment by robertohuertasm** [comment]
That gray should be ok 😊

**Comment by Hacksore** [comment]
@robertohuertasm ready for a review again.

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -5131,7 +5131,7 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
     },
     {
-      icon: 'zeit',
+      icon: 'vercel',
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedExtensions.ts:5134]: you must also change the name of the light icon

File: src/iconsManifest/supportedFolders.ts [hunk 2]
```diff
@@ -511,6 +511,11 @@ export const extensions: IFolderCollection = {
       extensions: ['.venv', '.virtualenv'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'vercel',
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedFolders.ts:515]: you still need to provide an opened version of the folder. Look for other folders in the extension to see how it's done.

**Review by robertohuertasm — CHANGES_REQUESTED** [review]

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Pull request merged**